src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public Color getRandomColor() {
+        return colors[random.nextInt(colors.length)];
+    }
```

Comment:
Better return color in String

Reply:
public Color getRandomColor() {
        return colors[random.nextInt(colors.length)];
    }
it will be string